deploy/role.yaml

```diff
@@ -28,6 +29,13 @@ rules:
   - deployments
   verbs:
   - "*"
+- apiGroups:
+  - apps
+  resources:
+  - replicasets
+  verbs:
+  - "get"
+  - "create"
```

Comment:
It's usually safer to add "list" as well if we add "get" just in case it's needed too.

Reply:
ok, I'll add that

Reply:
done in d47c571